src/main/target/NAZE/target.h

```diff
@@ -74,36 +82,66 @@
 #define USE_FLASH_M25P16
 
 #define GYRO
+#ifdef NAZE_GENERIC
 #define USE_GYRO_MPU3050
+#endif
+#if defined(NAZE_GENERIC) || defined(NAZE_rev5)
 #define USE_GYRO_MPU6050
+#endif
+#if defined(NAZE_GENERIC) || defined(NAZE_rev6)
 #define USE_GYRO_MPU6500
 #define USE_GYRO_SPI_MPU6500
+#endif
+
 
+#ifdef NAZE_GENERIC
 #define GYRO_MPU3050_ALIGN      CW0_DEG
+#endif
+#if defined(NAZE_GENERIC) || defined(NAZE_rev5)
 #define GYRO_MPU6050_ALIGN      CW0_DEG
+#endif
+#if defined(NAZE_GENERIC) || defined(NAZE_rev6)
 #define GYRO_MPU6500_ALIGN      CW0_DEG
+#endif
 
 #define ACC
+#if defined(NAZE_GENERIC)
 #define USE_ACC_ADXL345
 #define USE_ACC_BMA280
 #define USE_ACC_MMA8452
+#endif
+#if defined(NAZE_GENERIC) || defined(NAZE_rev5)
```

Comment:
My bad. I had not realised at that point that your aim is to make NAZE_GENERIC support all boards, including revs 5 and 6.